src/shared/components/checkbox/Checkbox.tsx

```diff
@@ -44,27 +48,43 @@ const isWithLabel = (props: CheckboxProps): props is WithLabelProps => {
   return 'label' in props;
 };
 
+const getThemeClasses = (theme: Theme = 'lighter') => {
+  if (theme === 'dark') {
+    return styles.themeDark;
+  }
+
+  if (theme === 'light') {
+    return styles.themeLight;
+  }
+};
```

Comment:
Would it be to horrible of me to ask that the default theme be "light" rather than "lighter"? It's just odd to that pale variant as the default.

Reply:
No it isn't 😁 

I chose that since it was the only theme that used the default styling at that time. Perhaps I can change the default styling to be light instead.

Reply:
✅